cat-vaadin-client/pom.xml

```diff
@@ -52,40 +51,26 @@
                 </exclusion>
             </exclusions>
         </dependency>
-
-        <!-- Added to provide logging output as Vaadin uses -->
-        <!-- the unbound SLF4J no-operation (NOP) logger implementation -->
-
         <dependency>
-            <groupId>org.springframework</groupId>
-            <artifactId>spring-web</artifactId>
+            <groupId>com.vaadin</groupId>
```

Comment:
Is this module good maintained (last commits etc.) ? Because in the past I had a lot of problems with dependencies like this that they are cancelled or not maintained for future versions. Is this given? Is this module maintained by the vaadin team itself?